modules/microshift-low-latency-tuned-auto.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-low-latency.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-low-latency-tuned-auto-activate_{context}"]
+= Automatically enable the {microshift-short} TuneD profile
+
+Included in the `microshift-low-latency` RPM package is a systemd service that you can configure to automatically enable a TuneD profile when the system starts. This ability is particularly useful if you are installing {microshift-short} in a large fleet of devices.
+
+.Prerequisites
+
+. You installed the microshift-low-latency RPM package on the host.
+. You enabled low latency in the {microshift-short} `config.yaml`.
+. You created a TuneD profile.
+. You configured the `microshift-baseline-variables.conf` file.
+
+.Procedure
+
+. Configure the `tuned.yaml` in the `/etc/microshift/` directory, for example:
++
+.Example tuned.yaml
+[source,yaml]
+----
+profile: microshift-baseline <1>
+reboot_after_apply: True <2>
+----
+<1> The `profile` value controls which TuneD profile is activated. In this example, the name of the profile is `microshift-baseline`.
+<2> The `reboot_after_apply` value controls whether the host must be rebooted after applying the profile. Valid values are `True` and `False`. For example, use the `True` setting to automatically restart the host after a new `ostree` commit is deployed.
++
+[IMPORTANT]
+====
+The host is restarted when the `microshift-tuned.service` runs, but it does not restart the system when a new commit is deployed. You must restart the host to enable a new commit, then the system starts again when the `microshift-tuned.service` runs on that boot and detects changes to profiles and variables.
+
+This double-boot can effect rollbacks. Ensure that you adjust the number of reboots in greenboot that are allowed before rollback when using automatic profile activation. For example, if 3 reboots are allowed before a rollback in greenboot, increase that number to 4.
```

Comment:
Do we have existing documentation that explains how to make this Greenboot change? If so, can we link to it here? The user should be told how to make this change.

Reply:
Other than this one nit about linking to docs to help the user know how to change the greenboot rollback count this LGTM.